internal/generate/packagemanifest/packagemanifest.go

```diff
@@ -17,37 +17,63 @@ package packagemanifest
 import (
 	"errors"
 	"fmt"
-	"io"
+	"io/ioutil"
 	"path/filepath"
 	"sort"
 
+	"sigs.k8s.io/yaml"
+
 	apimanifests "github.com/operator-framework/api/pkg/manifests"
 	"github.com/operator-framework/api/pkg/validation"
 	log "github.com/sirupsen/logrus"
 
 	genutil "github.com/operator-framework/operator-sdk/internal/generate/internal"
-	"github.com/operator-framework/operator-sdk/internal/generate/packagemanifest/bases"
 )
 
 const (
 	// File extension for all PackageManifests written by Generator.
 	packageManifestFileExt = ".package.yaml"
 )
 
+// GPkgManenerator is an implementation of the Generator interface
+type PkgManGenerator struct{}
```

Comment:
This shouldn't be exported because it isn't used anywhere
```suggestion
// generator implements the Generator interface
type generator struct{}
```